### ISSUE TITLE
MSMT17数据集

### ISSUE THREAD

**Issue by Weiziwei521** [JDAI-CV/fast-reid#373]
采用v1版本（train、test、list_gallery.txt、list_query.txt、list_train.txt、list_val.txt），将dataset_dir修改为本地路径之后，会出现错误AssertionError：Dataset folder not found。在msmt_v1中，没有main_dir啊，这该怎么办啊，谢谢您
配置文件为：
CUDNN_BENCHMARK: true
DATALOADER: {NAIVE_WAY: true, NUM_INSTANCE: 16, NUM_WORKERS: 8, PK_SAMPLER: true}
DATASETS:
  COMBINEALL: false
  NAMES: [MSMT17]
  TESTS: [MSMT17]
INPUT:
  CJ: {BRIGHTNESS: 0.15, CONTRAST: 0.15, ENABLED: false, HUE: 0.1, PROB: 0.8, SATURATION: 0.1}
  DO_AUGMIX: false
  DO_AUTOAUG: true
  DO_FLIP: true
  DO_PAD: true
  FLIP_PROB: 0.5
  PADDING: 10
  PADDING_MODE: constant
  REA:
    ENABLED: true
    MEAN: [123.675, 116.28, 103.53]
    PROB: 0.5
  RPT: {ENABLED: false, PROB: 0.5}
  SIZE_TEST: [384, 128]
  SIZE_TRAIN: [384, 128]
MODEL:
  BACKBONE: {DEPTH: 50x, FEAT_DIM: 2048, LAST_STRIDE: 1, NAME: build_resnet_backbone,
    NORM: BN, PRETRAIN: true, PRETRAIN_PATH: /media/speech/7672ffbf-7370-4075-b916-9fed437ab2cb/daima/fast-reid-master/fastreid/modeling/resnet50-ibn.pth,
    WITH_IBN: false, WITH_NL: false, WITH_SE: true}
  DEVICE: cuda:0
  FREEZE_LAYERS: [backbone]
  HEADS: {CLS_LAYER: circleSoftmax, EMBEDDING_DIM: 0, MARGIN: 0.35, NAME: EmbeddingHead,
    NECK_FEAT: after, NORM: BN, NUM_CLASSES: 0, POOL_LAYER: gempool, SCALE: 64, WITH_BNNECK: true}
  LOSSES:
    CE: {ALPHA: 0.2, EPSILON: 0.1, SCALE: 1.0}
    CIRCLE: {ALPHA: 128, MARGIN: 0.25, SCALE: 1.0}
    FL: {ALPHA: 0.25, GAMMA: 2, SCALE: 1.0}
    NAME: [CrossEntropyLoss, TripletLoss]
    TRI: {HARD_MINING: true, MARGIN: 0.0, NORM_FEAT: false, SCALE: 1.0}
  META_ARCHITECTURE: Baseline
  PIXEL_MEAN: [123.675, 116.28, 103.53]
  PIXEL_STD: [58.395, 57.120000000000005, 57.375]
  WEIGHTS: ''
OUTPUT_DIR: logs/msmt17/sbs_R50
SOLVER:
  AMP_ENABLED: false
  BASE_LR: 0.01
  BIAS_LR_FACTOR: 1.0
  CHECKPOINT_PERIOD: 30
  DELAY_ITERS: 30
  ETA_MIN_LR: 7.7e-07
  FREEZE_ITERS: 10
  GAMMA: 0.1
  HEADS_LR_FACTOR: 1.0
  IMS_PER_BATCH: 64
  MAX_ITER: 90
  MOMENTUM: 0.9
  OPT: SGD
  SCHED: WarmupCosineAnnealingLR
  STEPS: [40, 80]
  SWA: {ENABLED: false, ETA_MIN_LR: 3.5e-06, ITER: 10, LR_FACTOR: 10.0, LR_SCHED: false,
    PERIOD: 2}
  WARMUP_FACTOR: 0.01
  WARMUP_ITERS: 10
  WARMUP_METHOD: linear
  WEIGHT_DECAY: 0.0005
  WEIGHT_DECAY_BIAS: 0.0005
TEST:
  AQE: {ALPHA: 3.0, ENABLED: false, QE_K: 5, QE_TIME: 1}
  EVAL_PERIOD: 30
  IMS_PER_BATCH: 128
  METRIC: cosine
  PRECISE_BN: {DATASET: Market1501, ENABLED: false, NUM_ITER: 300}
  RERANK: {ENABLED: false, K1: 20, K2: 6, LAMBDA: 0.3}
  ROC_ENABLED: false

**Comment by L1aoXingyu** [comment]
![image](https://user-images.githubusercontent.com/19245504/103211251-a764c380-4942-11eb-99fd-a20f6e394267.png)

你把里面的路径改成 `train` 和 `test` 就行了，如果你的数据结构不一样，你可以自己注册一个新的数据集。
